core/src/scene/importer.cpp

```diff
@@ -188,7 +188,11 @@ void Importer::addSceneYaml(const Url& sceneUrl, const char* sceneYaml, size_t l
     auto& sceneNode = m_sceneNodes[sceneUrl];
 
     try {
+#ifdef TANGRAM_WINDOWS
```

Comment:
is this because vcpkg's yaml_cpp is using different version? This will break non-vcpkg builds.

Reply:
Done